CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+### Added
+
+- Added `Widget.disabled` https://github.com/Textualize/textual/pull/1785
```

Comment:
Is it worth adding something under "Changed" about `Button.disabled` or do you think it makes no sense?

Reply:
I don't think it really makes sense because there is still a `Button.disabled`.